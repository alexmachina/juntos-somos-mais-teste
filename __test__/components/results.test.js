import React from "react";
import Results from "../../pages/customers/list/components/results/index";
import renderer from "react-test-renderer";

describe("Components", () => {
  describe("Results", () => {
    it("Renders correctly", () => {
      const tree = renderer.create(<Results />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
