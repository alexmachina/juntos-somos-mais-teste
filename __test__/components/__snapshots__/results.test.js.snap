// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Results Renders correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-results-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
  />
</div>
`;
